app/README.md

```diff
@@ -29,3 +29,6 @@ needs to be queried to get older stats. As a result, the `get_specific_template_
 `template/rest.py` relies on a method, `fetch_template_usage_for_service_with_given_template()`, that first
 queries the `ft_notification_status` table and then the `notifications` table if the user needs to get stats
 from today.
+
+## Definition of message_limit
+The service message_limit field defines the limit of the number of messages the user can send per day. In the POST request above, it is set to 100. The service called "BVA" was created in the staging environment to update the message_limit of all services from 1000 to 100. The service is called, the message limit is checked, and the service is sent. This service was created in Postman, and the daily message limit is required when creating a service, so now the default message_limit is 100.
```

Comment:
Do we really need all the details around `BVA` for message limits? Can we just list that the `message_limit` in the system was recently updated to default to 100? Or an example of creating a new service in Postman but setting that `message_limit` to 100?